mappings/net/minecraft/item/BlockItem.mapping

```diff
@@ -1,29 +1,46 @@
 CLASS bae net/minecraft/item/BlockItem
 	FIELD a block Lbmm;
+	METHOD <init> (Lbmm;Lbbw$a;)V
+		ARG 1 block
+		ARG 2 settings
 	METHOD a place (Lbaf;)Lahd;
-	METHOD a setBlockState (Lbaf;Lbvk;)Z
-	METHOD a copyItemTagToBlockEntity (Lbhi;Lavy;Lev;Lbcb;)Z
+		ARG 1 context
+	METHOD a place (Lbaf;Lbvk;)Z
+		ARG 1 context
+		ARG 2 state
+	METHOD a writeTagToBlockEntity (Lbhi;Lavy;Lev;Lbcb;)Z
 		ARG 0 world
 		ARG 1 player
 		ARG 2 pos
 		ARG 3 stack
 	METHOD a getPlaceSound (Lbvk;)Lyj;
-	METHOD a addProperty (Lbvk;Lbwn;Ljava/lang/String;)Lbvk;
+		ARG 1 state
+	METHOD a (Lbvk;Lbwn;Ljava/lang/Comparable;)Lbvk;
```

Comment:
Is this normal ?

Reply:
It is a synthetic method with mapped parameter `value`:

```java
.<BlockState>map(value -> (BlockState)((AbstractPropertyContainer<Object, Object>)state).<T, Comparable>with(property, value))
```